issue_03/script.js

```diff
@@ -0,0 +1,18 @@
+const ul = document.getElementById('js-ul');
+const li = document.createElement('li');
+const a = document.createElement('a');
+const img = document.createElement('img');
+
+a.setAttribute('href', 'a1.html');
+a.textContent = 'a1';
+img.setAttribute('src', '/img/bookmark.png');
+a.insertAdjacentElement('afterbegin', img);
+ul.appendChild(li).appendChild(a);
+
+const cloneLi = li.cloneNode(true);
+const cloneA = cloneLi.firstElementChild;
+const nodeListOfCloneA = cloneA.childNodes;
+
+nodeListOfCloneA[0].src = '/img/message.png';
+nodeListOfCloneA[1].textContent = 'a2';
+li.after(cloneLi);
```

Comment:
I did not know the "cloneNode" !!:eyes: 

Q and IMO
I felt that if I wanted to repeat the same thing 10 times, I would write the same description 10 times if I used this method. I thought it would be better to use a more efficient method.
(I do not know this method well so I would like to know your opinion! )